solidity/contracts/LinkToken.sol

```diff
@@ -1,3 +0,0 @@
-pragma solidity ^0.4.23;
-
-import "linkToken/contracts/LinkToken.sol";
```

Comment:
The migrations still rely on this file. This might be passing CI because it's still in the cache.

https://github.com/smartcontractkit/chainlink/blob/chores/geth-startup-leveldb-closed/solidity/migrations/2_link_token_contract.js

Can we see if we're green after `rm -rf solidity/build/`?

Reply:
Removed build folder and re-ran `internal/ci/truffle_test`. All contracts compile and tests still pass.